add rg variable to cd template

kiwiidb opened the pull request:
Hi there!

I was following your excellent tutorial here:
https://cloudblogs.microsoft.com/opensource/2018/11/27/tutorial-azure-devops-setup-cicd-pipeline-kubernetes-docker-helm

When trying to deploy the CD pipeline, something went wrong: the az cli wasn't able to login. After some digging I found that this is because you named your Azure resource group the same as your cluster. I'd suggest this change and also an update in the tutorial where you add an extra variable `rg` to the pipeline.

kiwiidb committed:
add rg variable to cd template

msftclas commented:
[![CLA assistant check](https://cla.opensource.microsoft.com/pull/badge/not_signed)](https://cla.opensource.microsoft.com/Azure/phippyandfriends?pullRequest=43) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.microsoft.com/Azure/phippyandfriends?pullRequest=43) before we can accept your contribution.<br/><br/>:x: kiwiidb [sign now](https://cla.opensource.microsoft.com/Azure/phippyandfriends?pullRequest=43)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.microsoft.com/check/Azure/phippyandfriends?pullRequest=43) it.</sub>

mathieu-benoit commented:
Good idea, thanks for the proposition! Reviewing this PR right now, thanks!

kiwiidb committed:
fix comments about variables to add

kiwiidb commented:
Thanks for the answer. There was a commit unrelated to this PR here but I have removed it now. I have added another commit to update some comments about the variables to add.
